apps/fluent-tester/src/TestComponents/Switch/consts.ts

```diff
@@ -0,0 +1,9 @@
+export const HOMEPAGE_SWITCH_BUTTON = 'Homepage_Switch_Button';
+export const SWITCH_TESTPAGE = 'Switch_TestPage';
+
+/* Text Switch 1 */
+export const SWITCH_TEST_COMPONENT = 'Switch_Test_Component';
+export const SWITCH_ACCESSIBILITY_LABEL = 'E2E testing Switch accessibility label';
+
+export const SWITCH_TOGGLE_ON = 'Switch_Toggle_On';
```

Comment:
These two constants should only be one:
`SWITCH_ON_PRESS`

Please see comment above